### DISCUSSION TITLE
Welcome to the OpenPNM Discussions Forum

### DISCUSSION THREAD

**Discussion by jgostick** [PMEAL/OpenPNM#2176] [Announcements]
Github added the Discussions feature earlier this year and it is *amazing*!  This allows us to finally separate bug reports from help requests, which keeps the issue list much cleaner (and less stressful!) [^1].  So please go ahead and ask your question.  One of the developers will try to answer it in a timely fashion.  Also feel free to answer any questions you see too.  Just like stackoverflow, anyone can answer a question.

[^1]: The best part is that we can transfer items back and forth, so if you think you have question, but it actually ends up being a bug, we can just move it!

**Comment by engphb** [comment]
Hello my friend, how are you? 
I didn't understand some equations in the following code example.
https://github.com/PMEAL/OpenPNM/blob/dev/examples/simulations/steady_state/continuum_heat_transfer.ipynb 

In line 5 you putted this equation: T_in = 30*np.sin(np.pi*pn['pore.coords'][inlets, 1]/5)+50

Could you explain to me the origin of this equation?

And in line 8 you putted this equation: soln = 30*np.sinh(np.pi*X/5)/np.sinh(np.pi/5)*np.sin(np.pi*Y/5) + 50

Could you explain to me the origin of this equation?

Regards.

**Reply by jgostick** [reply]
This is just inputting a sinusoidal boundary condition for the temperature, and the second one is the analytical solution for this case.  This is just standard solutions from a conduction heat transfer textbook.

**Reply by engphb** [reply]
Could you give me the name of this book?

**Reply by jgostick** [reply]
I don't remember, but any book on heat conduction will have the solution to 2D conduction in a plane.

**Reply by engphb** [reply]
Ok professor, thank you.
